jazz_deployments/components/config.js

```diff
@@ -0,0 +1,48 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
```

Comment:
please use => arrow functions throughout the code. I am seeing some inconsistency in the code where some place arrow is used and other place 'function'.

Reply:
Using arrow functions throughout the code